pywsitest/rest_request.py

```diff
@@ -0,0 +1,103 @@
+import requests
+
+
+class RestRequest:
+    """
+    A class representing a rest request
+
+    Attributes:
+        uri (str)
+        method (str)
+        headers (dict)
+        body (dict)
+        delay (float)
+
+    Methods:
+        with_header(key (str), value (str)):
+            Adds a header and returns RestRequest
+        with_body(body (dict)):
+            Adds a body and returns RestRequest
+        with_delay(delay (float)):
+            Adds a delay and returns RestRequest
+        send(timeout (float)):
+            Composes and sends rest request, returning request response
+
+    Usage:
+        rest_request = (
+            RestRequest("https://example.com", "POST")
+            .with_header("Authorization", "jwt_token")
+            .with_body({"abc": 123})
+            .with_delay(10.0)
+        )
+
+        rest_request.send(10.0)
```

Comment:
Could you edit this so that the delay is less than the timeout being passed to send - I like to keep any examples in a working state